Flippers/src/main/java/io/doist/recyclerviewext/flippers/ViewExt.kt

```diff
@@ -0,0 +1,9 @@
+package io.doist.recyclerviewext.flippers
+
+import android.view.View
+
+inline var View.isVisible: Boolean
+    get() = visibility == View.VISIBLE
+    set(value) {
+        visibility = if (value) View.VISIBLE else View.GONE
+    }
```

Comment:
It will lead that IDE shows multiple options to import `View.isVisible` for projects that have both `RecyclerViewExtension` and `core-ctx` :) May be we can remove it or make it internal at least?

Reply:
Made it internal, nice catch! 👌